udaykalra/portfolio/src/main/webapp/index.html

```diff
@@ -31,27 +35,40 @@ <h1>Uday Kalra</h1>
     <div class="container-fluid main-content">
       <div class="row topRow center">
         <div class="col-sm-4 center">
-          <img src="face2.jpg" class="rounded-circle center profile_img" alt="..." />
+          <img
+            src="face2.jpg"
+            class="rounded-circle center profile_img"
+            alt="..."
```

Comment:
Did you want to provide an accessibility string?
https://google.github.io/styleguide/htmlcssguide.html#Multimedia_Fallback

Also applies on line 70 below.